sound/soc/sof/ops.h

```diff
@@ -100,6 +100,14 @@ static inline int snd_sof_dsp_post_fw_run(struct snd_sof_dev *sdev)
 	return 0;
 }
 
+/* parse platform specific extended manifest */
```

Comment:
can you please condense the commit message to " add parse_platform_ext_manifest op to parse platform-specific config data"